packages/v4/src/content/design-guidelines/components/number input/number-input.md

```diff
@@ -0,0 +1,57 @@
+---
+id: Number input
+section: components
+---
+
+A **number input** provides a quick and effective way for users to enter and modify a numeric value. Similar to <a href="https://www.patternfly.org/v4/components/input-group/">input groups</a>, number inputs combine two input types into one control: A text input field with buttons on either side. When combined, these elements allow users to input numeric values in a responsive and scalable way.
+
+## Elements
+<img src="./img/number_input_anatomy.png" alt="Number input elements" width="537px"/>
+
+1. **Label:** Indicates what value the user should input in the numeric field.
+2. **Input field:** Area for the user to type in their numeric input.
+3. **Buttons:** Increase or decrease the value by set increments.
+4. **Helper text:** Explains or provides additional context that helps the user provide the right information. This is an optional field.
+5. **Unit:** Provides a unit symbol for the value in the input field, when applicable. This is an optional field.
+
+## Usage
+**Use a number input when:**
+* Users need a quick and precise way to adjust values within defined parameters.
+* Users need to make small incremental changes that require few clicks.
+* You need to bypass device limitations, such as keyboardless interactions and form submissions on mobile.
+
+**Don’t use a number input when:**
+* The numeric value range is infinite.
+* The numeric value can be either a fractional or whole number.
+* The numeric value required is not part of a range, or is arbitrary. In this use case, use an <a href="https://www.patternfly.org/v4/components/input-group/">input group</a> instead.
+* There is not enough space in the UI to account for the interaction between the input field and the buttons.
+
+**When using number inputs, follow these best practices:**
+* Display a default numeric value in the input field that users can modify by typing directly into the input field or scale using the increase or decrease buttons.
+* Keep labels concise and specific to effectively communicate what value the user should enter into the input field.
+* Determine an increment size relative to your specific value range. Scale your increment size to accommodate for the types of values users will input in context.
```

Comment:
Can you include the example we talked about where you just say something like "For example, if you range is from 0-10, have increments of 1, but if your range is from 0-100, have increments of 10"

Reply:
@kevinhatchoua think you might've missed this comment as well!